bin/contact-search-backfill

```diff
@@ -1,12 +1,11 @@
 #!/usr/bin/env python
-import click
 from gevent import monkey
 
 monkey.patch_all()
 import gevent_openssl
 
 gevent_openssl.monkey_patch()
-
+import click
```

Comment:
Could we have a newline above this import (and above `import gevent_openssl`)? This applies in a few other cases within this commit as well.